app/cdap/components/WranglerGrid/SelectColumnPanel/types.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { IStatistics, IParams } from 'components/GridTable/types';
+
+export interface IRecords {
+  wid?: string;
+  payload?: IParams;
+  body?: string;
+  path?: string;
+  canBrowse?: boolean;
+  name?: string;
+}
+
+export interface IHeaderNamesList {
+  name: string;
+  label: string;
+  type: string[];
+}
+
+export interface IAddTransformationProps {
+  transformationDataType: string[];
+  transformationName: string;
+  columnsList: IHeaderNamesList[];
+  missingItemsList: IStatistics;
+  onCancel: () => void;
+}
+
+export interface IMultipleSelectedFunctionDetail {
+  value: string;
```

Comment:
Again if this is a known list of 5 values, this should be those five values instead of all possible strings as the type.

Reply:
Done

Reply:
we have added only the know list of values as the type of this key instead of string type